docs/sources/configure/_index.md

```diff
@@ -2281,6 +2281,29 @@ bloom_shipper:
     # the tasks above this limit will fail an error.
     # CLI flag: -bloom.shipper.blocks-downloading-queue.max_tasks_enqueued_per_tenant
     [max_tasks_enqueued_per_tenant: <int> | default = 10000]
+
+  blocks_cache:
+    # Whether embedded cache is enabled.
+    # CLI flag: -blocks-cache.enabled
+    [enabled: <boolean> | default = false]
+
+    # Maximum memory size of the cache in MB.
+    # CLI flag: -blocks-cache.max-size-mb
+    [max_size_mb: <int> | default = 100]
```

Comment:
Unfortunately we cannot use a better default, since the config is used in multiple places :(

Reply:
we can, but I do not know what is suitable value here... it depends on the customer, and how much data they have, and it's important if they have enough disk space...  let's come back to this question maybe later ...